lib/ipmi_channel.c

```diff
@@ -182,7 +182,7 @@ _ipmi_set_channel_access(struct ipmi_intf *intf,
 	memset(&data, 0, sizeof(data));
 	data[0] = channel_access.channel & 0x0F;
 	data[1] = (access_option << 6);
-	if (channel_access.alerting) {
+	if (!channel_access.alerting) {
```

Comment:
This change is completely wrong and just breaks everything. Please see carefully how and where this structure field is used.

Reply:
The bug is, nonetheless, present indeed. It needs to be fixed elsewhere though. I'll do it.